source/index.txt

```diff
@@ -66,9 +68,29 @@ core operations. This section contains information on how to:
 API Documentation
 -----------------
 
-The `API documentation <{+api+}/apidocs/mongodb-driver-sync/>`__
-has detailed technical information about classes, methods, and
-configuration objects within the MongoDB Java driver.
+The MongoDB Java driver API documentation is split into several sections.
+Each section has detailed technical information about classes, methods, and
+configuration objects within the MongoDB Java driver:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 35 65
+
+   * - Section
+     - Description
+
+   * - `BSON <{+api+}/apidocs/bson/index.html>`__ 
+     - Contains the base BSON classes
```

Comment:
Nitpick: you probably don't need "Contains the", especially if you change the header of this table from "Section" to library or package as requested in a comment further up.